src/IPAddr.cc

```diff
@@ -126,6 +103,37 @@ void ConnKey::Init(const IPAddr& src, const IPAddr& dst, uint16_t src_port, uint
     transport = proto;
 }
 
+session::detail::Key ConnKey::SessionKey() const {
+    uint8_t* temp = new uint8_t[PackedSize()];
```

Comment:
`PackedSize()` never returns a different value here. I realize you mentioned storing the result of `Pack` in the future in the commit message, but removing the added calls to `PackedSize` is an easy one.